azure-pipelines/release-extension.yml

```diff
@@ -27,6 +27,19 @@ parameters:
     type: boolean
     default: false
 
+  # A list of publish commands to run. It defaults to publishing a single .vsix file. If you want to publish multiple .vsix files (platform specifc), you can add mutliple commands to this list.
+  - name: publishCommands
+    type: object
+    default:
+      [
+        "vsce publish --azure-credential --packagePath $(vsixFileName) --manifestPath extension.manifest --signaturePath extension.signature.p7s",
+      ]
+
+  # When true, skips check to make sure there are not multiple .vsix files found in the build directory
+  - name: publishMultipleVsixFiles
```

Comment:
Can we just remove the check rather than needing to add a parameter for just it?

Reply:
I agree with Brandon. If the pipeline supports building/publishing single and multiple vsix at the same time we can just remove the check.